ied-be/src/templates/certificates/perm/certificate.css

```
@@ -0,0 +1,396 @@
+  position: relative;
+  width: var(--page-width);
+  height: var(--page-height);
+  padding: var(--page-padding);
+  overflow: hidden;
+  background: linear-gradient(
+    135deg,
+    var(--paper),
+    var(--paper-warm) 52%,
+    var(--paper)
+  );
+  page-break-after: always;
```

Stylelint (17.9.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)